Documentation/Update/Index.rst

```diff
@@ -0,0 +1,128 @@
+.. include:: ../Includes.txt
+
+.. highlight:: bash
+
+.. _update:
+
+==================
+TYPO3 Patch Update
+==================
+
+
+.. _typo3-version-basics:
+
+TYPO3 Version Basics
+====================
+
+The TYPO3 version numbers consist of the following parts:
+
+<major>.<minor>.<patch>
+
+For example 9.5.1 is major version 9, minor version 5 and patch version 1.
+
+* **major**: An incrementation in a major version (e.g. 8.7.23 -> 9.5.5) may mean breaking
+  changes and requires an extensive update procedure.
+* **minor**: A change in a minor version should contain no deprecation. This means the minor
+  version is backwards compatible (e.g. 6.0.0 -> 6.2.0). The new minor version may
+  introduce new features and bugfixes.
+* **patch**: A change in the patch number will only contain bugfixes, no deprecations and no
+  new features (e.g. 9.5.1 -> 9.5.2)
+
+For more information about deprecations, see :ref:`handling-deprecations`.
```

Comment:
The reference points to a sentence `Minor versions are always backwards compatible` which is not true. TYPO3 does not use semantic versioning. Even security patches and bug fixes might introduce breaking changes.

Reply:
@DanielSiepmann  Do you have a link where it is explained how TYPO3 version numbers are used?

Reply:
I couldn't find an official documentation, but found these references:

* https://typo3.org/article/embrace-and-innovate-typo3-cms-7/
* https://typo3.org/article/voting-for-the-new-release-cycles-for-typo3-cms/
* https://wiki.typo3.org/Blueprints/Release
* https://typo3.org/article/voting-for-the-new-release-cycles-for-typo3-cms/
* https://typo3.org/article/the-typo3-community-publishes-typo3-cms-70/
* https://typo3.org/cms/roadmap/
* https://typo3.org/article/typo3-release-improvements-v8-maintenance-release-schedule/

Reply:
@DanielSiepmann I removed the information about the versions from this patch and removed everything that was not absolutely necessary for main intention. The main intention was: Currently there is only one chapter "Upgrade" which handles a number of steps, most of which are not required for patch updates. It makes updating look like a huge and daunting task, which it is usually not for patch updates. I would like to point out difference between major update + patch update and provide a sample workflow for updating patch updates.

If this is helpful, we can use it. If not, we should close PR and rethink.